### ISSUE TITLE
Main navigation uses images of text

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#13]
The main nav menu should use icons + text, rather than including the text within the icon.

**Comment by NomeQ** [comment]
Main navigation is now missing entirely. We still need Maps & Schedules, Fares, etc

**Comment by NomeQ** [comment]
Same
